public/pages/PolicyDetails/components/PolicySettings/PolicySettings.tsx

```diff
@@ -43,23 +43,18 @@ export default class PolicySettings extends Component<PolicySettingsProps, Polic
     }
   }
 
-  // TODO: Needs to be updated to handle template array
   getTemplates = (ismTemplates: ISMTemplate[]): ISMTemplate[] => {
-    const templateArray = [];
-    ismTemplates.map((ismTemplate)=> {
-      templateArray.push({
-        indexPatterns: ismTemplate.index_patterns,
-        priority: ismTemplate.priority,
-      })
-    })
+    const templateArray = ismTemplates.map(ismTemplate => ({
```

Comment:
Previously this was renaming it to indexPatterns, but if it's not then what is the purpose of this function? Seems to take in a list of ISMTemplates and returns a list of ISMTemplates in the same format.

Reply:
Good call. Removing this and simplifying.